application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -144,11 +137,10 @@ public void onSlashCommand(@NotNull SlashCommandInteractionEvent event) {
     private boolean handleChecks(@NotNull Member bot, @NotNull Member author,
             @Nullable Member target, @NotNull IReplyCallback event) {
         // Member doesn't exist if attempting to audit a user who is not part of the guild.
-        if (target != null && !ModerationUtils.handleCanInteractWithTarget(ACTION_VERB, bot, author,
-                target, event)) {
+        if (target == null) {
             return false;
         }
-        return ModerationUtils.handleHasAuthorRole(ACTION_VERB, hasRequiredRole, author, event);
+        return ModerationUtils.handleCanInteractWithTarget(ACTION_VERB, bot, author, target, event);
```

Comment:
You changed the logic here, this is incorrect. Right now you removed the ability to audit users who left the guild, but the command is supposed to support that.

Reply:
```java
return !(target != null && !ModerationUtils.handleCanInteractWithTarget(ACTION_VERB, bot, author, target, event))
```
will this work?

Reply:
Whatever that is, it is super unreadable. Instead I would do:
```java
// Auditing users outside of the guild always works
if (target == null) {
  return true;
}

return ModerationUtils.handleCanInteractWithTarget(...);
```

Reply:
> Whatever that is, it is super unreadable.

agree